frontend/src/components/panels/LayerTree.vue

```diff
@@ -346,22 +357,26 @@ export default defineComponent({
 				layer.entry.layer_metadata.selected = false;
 			});
 		},
-		closest(tree: HTMLElement, clientY: number): { insertFolder: BigUint64Array; insertIndex: number; insertAboveNode: Node } {
+		closest(tree: HTMLElement, clientY: number): DraggingData {
```

Comment:
A more descriptive name for `closest()` would also be pretty helpful, I'm a bit confused what this does overall based on its name.